tests/e2e/specs/TextAnnotation.spec.js

```diff
@@ -44,18 +40,54 @@ describe('Text Annotation', () => {
 
     connectNodesWithFlow('association-flow-button', textAnnotationPosition, taskPosition);
 
-    cy.get('[data-test=save-process]').click();
+    const expectedXML = `<bpmn:process id="Process_1" isExecutable="true">
+    <bpmn:laneSet>
+      <bpmn:lane id="node_3" name="">
+        <bpmn:flowNodeRef>node_1</bpmn:flowNodeRef>
+        <bpmn:flowNodeRef>node_7</bpmn:flowNodeRef>
+      </bpmn:lane>
+      <bpmn:lane id="node_4" name="" />
+      <bpmn:lane id="node_5" name="" />
+    </bpmn:laneSet>
+    <bpmn:startEvent id="node_1" name="Start Event" />
+    <bpmn:task id="node_7" name="Task" />
+    <bpmn:textAnnotation id="node_6">
+      <bpmn:text>New Text Annotation</bpmn:text>
+    </bpmn:textAnnotation>
+    <bpmn:association id="node_8" associationDirection="None" sourceRef="node_6" targetRef="node_7" />`;
+
+    cy.get('[data-test=downloadXMLBtn]').click();
+    cy.window()
+      .its('xml')
+      .then(xml => xml.trim())
+      .then(xml => {
+        expect(xml).to.contain(expectedXML.trim());
+      });
+  });
```

Comment:
This change adds something we can assert about outside of core.

Reply:
We have a function called `assertDownloadedXmlContainsExpected` in `utils.js`, could that be used here?